src/posix/platform/trel.cpp

```diff
@@ -62,11 +63,16 @@ typedef struct TxPacket
     otSockAddr       mDestSockAddr;
 } TxPacket;
 
+class Counters : public otPlatTrelCounters, public ot::Clearable<Counters>
```

Comment:
Suggest using `memset()` and not use `ot::` (core-internal) types.

Reply:
Done